### PR TITLE
feat: Use icons for text align property

### PR DESCRIPTION
**Pull request by nschnierer**
Bildschirmfoto 2020-11-03 um 13 12 53

See #2201

### CODE REVIEW

File: src/actions/actionProperties.tsx [hunk 1]
```diff
@@ -39,6 +39,9 @@ import {
   SloppinessArchitectIcon,
   SloppinessArtistIcon,
   SloppinessCartoonistIcon,
+  TextAlignLeft,
+  TextAlignCenter,
+  TextAlignRight,
```
Comment (lipis) [inline src/actions/actionProperties.tsx:44]: maybe add `Icon` for consistency with other icons

**Comment by dwelle** [comment]
:+1:

I'd align the size with rest of the icons, though:

image

**Comment by lipis** [comment]
@dwelle I think they are properly aligned with the rest of the icons there.. eventually we'll change the rest of font related ones to icons as well and everything will be aligned :)
Screenshot 2020-12-03 at 7 53 40 PM

**Comment by dwelle** [comment]
Alright, let's keep it.

One thing though, changing to the svg/button or whatever it is now introduced a regression: when you're editing text and change text align, it now confirms (exits) the editor.

**Comment by steveruizok** [comment]
image

Whimsical has a pretty good UI for font size / text alignment. Would something like this work for Excalidraw? (Font family would break the mold a bit.)

**Review by lipis — APPROVED** [review]

**Comment by dwelle** [comment]
Before merging, we still need to fix the UX regression: https://github.com/excalidraw/excalidraw/pull/2379#issuecomment-738182782

**Comment by lipis** [comment]
@dwelle  I was just about to comment on that..!

**Review by lipis — CHANGES_REQUESTED** [review]
> One thing though, changing to the svg/button or whatever it is now introduced a regression: when you're editing text and change text align, it now confirms (exits) the editor.

**Comment by dwelle** [comment]
Closing in favor of https://github.com/excalidraw/excalidraw/pull/2857

**Pull request closed without merging**